File: Difficulty/Easy/[204] Count Primes/solution.py
class Solution:
    def countPrimes(self, n: int) -> int:
        isPrim, count = [1] * n, 0
        for i in range(2, n):
            if isPrim[i]:
                for j in range(i*i, n, i):
                    isPrim[j] = 0
        for j in range(2, n):
            if isPrim[j]:
                count += 1
        return count

File: Difficulty/Easy/[204] Count Primes/solution.cpp
class Solution {
public:
    int countPrimes(int n) {
        vector<bool> check(n, true);
        for (int i = 2; i * i < n; i++) {
            if (check[i]) {
                for (int j = i * i; j < n; j += i){
                    check[j] = false;
                }
            }
        }
        int count = 0;
        for (int i = 2; i < n; i++) {
            if (check[i])
                count++;
        }
        return count;
    }
};